numba_dpex/core/kernel_interface/func.py

```diff
@@ -113,6 +140,20 @@ def cache_hits(self):
         """Cache hit count accessor"""
         return self._cache_hits
 
+    def _strip_usm_metadata(self, argtypes):
```

Comment:
Same with this function, can be moved to a common place and reused.

Reply:
Moved to separate caching_utils.py